apps/ai-content-generator/src/configs/features/featureConfig.tsx

```diff
@@ -0,0 +1,44 @@
+import { FeatureConfig } from './featureTypes';
+import translatePrompt from '@configs/prompts/translatePrompt';
+import contentPrompt from '@configs/prompts/contentPrompt';
+import seoKeywordsPrompt from '@configs/prompts/seoKeywordsPrompt';
+import seoDescriptionPrompt from '@configs/prompts/seoDescriptionPrompt';
+
+enum AIFeature {
+  TITLE = 'title',
+  CONTENT = 'content',
+  TRANSLATE = 'translate',
+  SEO_DESCRIPTION = 'seoDescription',
+  SEO_KEYWORDS = 'seoKeywords',
+}
+
+const featureConfig: FeatureConfig = {
+  [AIFeature.TITLE]: {
+    title: 'Title',
+    helpText: 'Help text',
+    Component: () => <div>title</div>,
```

Comment:
This will be replaced with a TitleGenerator component in the next PR, showing off custom components